remote-api/src/main/java/org/alfresco/rest/api/impl/AspectsImpl.java

```diff
@@ -125,32 +138,50 @@ public Aspect getAspectById(String aspectId)
         if (aspectDefinition == null)
             throw new EntityNotFoundException(aspectId);
 
-        return this.convertToAspect(aspectDefinition);
+        return this.convertToAspect(aspectDefinition, ALL_PROPERTIES);
     }
 
-    public Aspect convertToAspect(AspectDefinition aspectDefinition)
+    public Aspect convertToAspect(AspectDefinition aspectDefinition, List<String> includes)
     {
-        List<PropertyDefinition> properties = this.classDefinitionMapper.fromDictionaryClassDefinition(aspectDefinition, dictionaryService).getProperties();
-        return new Aspect(aspectDefinition, dictionaryService, properties);
+        try
+        {
+            Aspect aspect = new Aspect(aspectDefinition, dictionaryService);
+            constructFromFilters(aspect, aspectDefinition, includes);
+            return aspect;
+        }
+        catch (Exception ex)
+        {
+            logger.warn("Failed to parse Aspect " + aspectDefinition.getName().toPrefixString());
```

Comment:
Swallowing errors at the API level is generally not a good idea, the user needs to know they made a mistake. What causes this error? It should be caught and re-thrown so the appropriate HTTP error code is returned.